src/model/migrations.js

```diff
@@ -489,6 +504,34 @@ export default async function runMigrations() {
 
   migrations.push(v12);
 
+  /*
+   *************** Migration v13 ******************
+   * Migrates from local storage to mmkv
```

Comment:
could the comment include "for hidden coins, pinned coins, savings toggle, and open families"

Reply:
looks like @benisgold included this both in v13 and v12. idk, I removed this from v12

Reply:
I'm not exactly sure what this is referring to, but it looks good now! v12 was used to migrate the format of hidden and pinned tokens since we now needed to support unique ID (which includes address and network, not just address, as there would then be duplicates across L2s)